src/marqo/tensor_search/models/search.py

```diff
@@ -63,9 +63,33 @@ class SearchContextTensor(BaseModel):
     vector: List[float]
     weight: float
 
+    def __init__(self, **data):
+        try:
+            super().__init__(**data)
+        except ValidationError as e:
+            raise InvalidArgError(message=e.json())
+
+class SearchContextDocumentsParameters(BaseModel):
+    tensorFields: Optional[List[str]] = None
+    excludeInputDocuments: Optional[bool] = True
```

Comment:
Should this be Optional[bool] = True or bool = True? Do you want to accept None here?